.github/pull_request_template.md

```diff
@@ -1,13 +1,16 @@
 ## Describe the behavior changes introduced in this PR
 
-## Linked Issues?
+<!-- Write this section if Pull Request has linked issues -->
+## Linked Issues
 
-resolves #<issue number>
+<!-- Use this if merging should auto-close an issue, one issue per line -->
+resolves #<!-- issue number -->
 
-related to #<issue number>
+<!-- Use this if merging should NOT auto-close an issue, one issue per line -->
+related to #<!-- issue number -->
 
 ## Testing Instructions
 
-Please include any additional commands or pointers in addition to our [standard PR testing process](/docs/Development.md#testing-pull-requests).
+Please include any additional commands or pointers in addition to our [standard PR testing process](https://pelorus.readthedocs.io/en/latest/Development/#testing-pull-requests).
```

Comment:
I don't thin this is right. The full URL won't be fine if used with $ mkdocs serve to test on a localhost.

Reply:
This file will not be served, right? (just the files in the `docs` folder)